backend/lambdas/generators/json_report/tests/test_generate_report.py

```diff
@@ -225,6 +226,27 @@
     end_time=datetime(year=2020, month=2, day=19, hour=15, minute=1, second=55, tzinfo=timezone.utc),
 )
 
+TEST_REPO_HEALTH = PluginResult(
+    plugin_name="Repo Health",
+    plugin_type="configuration",
+    success=True,
+    details={
+        "repo_health": [
```

Comment:
What benefit is having the plugin name be a key in the plugin details bringing?

Reply:
Bringing everything together:

> But does it make sense to be a dictionary? Is it serving the same purpose as with inventory plugins?

I'm not sure what the intent was with `inventory` (the docs are kinda sparse). I interpreted it as a way to segment things within the inventory type. With that interpretation, it makes sense to me to have the same thing for `configuration`, as handling `Github Repo Health` and a hypothetical  `AWS Configuration` plugin would be completely different.

I'm not attached to doing it this way though. What was the intent of the dictionary with the `inventory` type?